### PR TITLE
HDRP/Fix package version showing package after the last "verified" package

### PR DESCRIPTION
**Pull request by RSlysz**
After enquiering on FB issue on package version shown in wizard not the same as the one shown in package manager windows, it seams that this is expected behaviour of package lifecycle v2.
Only remaining :
- a typo (fixed in this PR)
- a design decision to make: align Wizard with Package Manager or remains with old behaviour for package last version discoverability

**UPDATE:**

Removed last package version check from wizard and add a button for shortchut check this in Package Manager. So now we have alignment.
image

### Tests done
Manually tested that:
- the version of current HDRP package appears. Tested with local package only. "(local)" should only appears at end for local packages.
- the retrieval of config package local installation still work. Changes introduced a regression on this due to asynchrone call conflict. It is handled with a stack so we should not bother on order but previous code only did it if nothing was in the stack. I dunno the reason. In the test, no local package was installed so it found it and display the installation button after the fix.
- the **Check update** button open the Package Manager window on the HDRP package. Also tested with the window already open. Not tested with the window opened on another package but this is package manager API so I assume it was tested on their side.

### CODE REVIEW

**Review by sebastienlagarde — APPROVED** [review]

**Comment by sebastienlagarde** [comment]
>a design decision to make: align Wizard with Package Manager or remains with old behaviour for package last version >discoverability
if telling people that there is a new package cause us so much pain (we have get fogbugzz issue because of that few times already), then let's simply remove this feature and maybe replace by a shortcut to packman and an info box saying that latest version can be found in packman windows?

**Comment by RSlysz** [comment]
**Convert back to draft:** 
I'll replace the version check with a link to package manager windows in this PR

**Comment by RSlysz** [comment]
**Convert back to draft:**
Missing documentation update

**Comment by RSlysz** [comment]
@JordanL8, @sebastienlagarde  I have seen that the documentation was not up to date. I updated it. Though a pass is needed on the DXR side. (VR have no change)

File: com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Wizard.md [hunk 1]
```diff
@@ -97,5 +97,6 @@ When upgrading a project from the built-in render pipeline to HDRP, you need to
 
 - **Upgrade Project Materials to High Definition Materials**: Upgrades every Material in your Unity Project to HDRP Materials.
 - **Upgrade Selected Materials to High Definition Materials**: Upgrades every Material currently selected to HDRP Materials.
+- **Upgrade HDRP Materials to Latest Version:** Upgrades every Materials in your Unity Project to latest version available.
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Wizard.md:100]: Upgrades every Material in your Unity Project to the latest version.

File: com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Wizard.md [hunk 2]
```diff
@@ -8,16 +8,15 @@ To open the **Render Pipeline Wizard**, go to **Window > Render Pipeline** and s
 
 ## Packages
 
-At the top of the window, there is an information box that shows you the currently installed version of HDRP, as well as the latest version of HDRP that is compatible with your current Unity version.
+At the top of the window, there is an information text that shows you the currently installed version of HDRP. The button **Check update** provide you a quick shortcut to the **Package Manager** windows onto the HDRP package.
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Wizard.md:11]: The **Check Update** button provides a shortcut to the HDRP window in the Package Manager.

**Review by JordanL8 — CHANGES_REQUESTED** [review]
Rewrote a couple of bits

File: com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Wizard.md (hunk 2 above)
Comment (RSlysz) [reply com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Wizard.md:11]: Well, not sure but now I understand that HDRP have a window but it is in fact Package Manager windows, with HDRP package selected in the package list on the left, so the HDRP package description is shown on the right.

Perhaps this?
The **Check Update** button provides a shortcut to the HDRP package in the Package Manager window.

Comment (JordanL8) [reply com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Wizard.md:11]: Yeah, sounds good

**Comment by iM0ve** [comment]
Please specify whats tested :) Its a bit more than a typo, so I think still worth running a wizard in an empty 3D project would be beneficial.

**Comment by RSlysz** [comment]
> Please specify whats tested :) Its a bit more than a typo, so I think still worth running a wizard in an empty 3D project would be beneficial.

Right sorry. I updated the PR with what was tested.

**Review by JordanL8 — APPROVED** [review]

**Review by iM0ve — APPROVED** [review]
Thanks for the update, looks good.

**Review by JulienIgnace-Unity — APPROVED** [review]

**Review by sebastienlagarde — APPROVED** [review]

**Pull request merged**